### PR TITLE
SUIT-9610 .acceptModal should accept empty string

### PR DESCRIPTION
**Pull request by andrea1gard**

### CODE REVIEW

File: lib/constants/validationKeys.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ const validationKeys = {
 	ST_VAR_OR_POSITIVE_NUMBER: Symbol('stVarOrPositiveNumberNotZero'),
 	ST_VAR_OR_NUMBER: Symbol('stVarOrNumber'),
 	NON_EMPTY_STRING_OR_UNDEFINED: Symbol('nonEmptyStringOrUndefined'),
-	NON_EMPTY_STRING_OR_NUll: Symbol('nonEmptyStringOrNull'),
+	STRING_OR_NULL: Symbol('StringOrNull'),
```
Comment (JeKachan) [inline lib/constants/validationKeys.js:10]: Symbol parameter should starts from lowecase

**Review by JeKachan — CHANGES_REQUESTED** [review]

**Review by JeKachan — APPROVED** [review]

**Review by Noorama — APPROVED** [review]

File: testDefinition/examplesTS/windowChain.ts [hunk 2]
```diff
@@ -12,6 +12,7 @@ async function windowTest() {
 	await window().goForward();
 	await window().dismissModal();
 	await window().acceptModal();
```
Comment (JeKachan) [inline testDefinition/examplesTS/windowChain.ts:14]: According to validation - acceptModal can't be called without arguments and may fall in runtime with validation error, please check it.

Comment (andrea1gard) [reply testDefinition/examplesTS/windowChain.ts:14]: sorry, just going back to this.
Do you mean in the `validateStringOrNull = (val, name)` function or somewhere else?

Comment (JeKachan) [reply testDefinition/examplesTS/windowChain.ts:14]: No, just need update type definition of this function

**Review by JeKachan — APPROVED** [review]

**Pull request merged**